queries.sql

```diff
@@ -9,3 +9,30 @@
  select * from animals where name !='Gabumon';
  select * from animals where weight_kg >=10.4 and weight_kg <=17.3;
 
+begin;
+update animals set species= 'unspecified';
+rollback;
+
+ begin;
+ update animals set species= 'digimon' where name like '%mon';
+ update animals set species='pokemon' where species is null;
+ commit;
```

Comment:
- [ ] According to the requirements, kindly write a query which verifies that the change was made and persists after the commit. e.g

```sql
 SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```